libs/dh/globalization/configuration-localization/src/lib/dh-transloco.module.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ModuleWithProviders,
+  NgModule,
+  Optional,
+  SkipSelf,
+} from '@angular/core';
+import { dhTranslocoHttpLoaderProvider } from '@energinet-datahub/dh/globalization/data-access-localization';
+import { environment } from '@energinet-datahub/dh/shared/environments';
+import {
+  TRANSLOCO_CONFIG,
+  translocoConfig,
+  TranslocoModule,
+} from '@ngneat/transloco';
+
+@NgModule({
+  imports: [TranslocoModule],
```

Comment:
question: is it necessary to import the `TranslocoModule` here when it's also imported in those modules where the `transloco` directive is used (e.g. `ShellModule`)?

Reply:
Yes, `TranslocoModule` exports the declarables.

I'm worried about it being a mixed module. It doesn't make sense in my head because it might introduce multiple instances of the Transloco dependencies for lazy loaded feature chunks. Let's stay on the lookout for this and submit an issue if it causes issues.